### PR TITLE
Adding DOFST, DOSTA and PARAD for Endurance

### PR DESCRIPTION
**Pull request by cwingard**

### CODE REVIEW

File: qartod/parad/parad_qartod_climatology_test_values.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+CE06ISSP,SP001,10-PARADJ000,parad_j_cspp_instrument,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-10-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE06ISSP,SP001,10-PARADJ000,parad_j_cspp_instrument_recovered,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-10-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE07SHSP,SP001,09-PARADJ000,parad_j_cspp_instrument,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-09-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE07SHSP,SP001,09-PARADJ000,parad_j_cspp_instrument_recovered,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-09-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
```
Comment (reedan88) [inline qartod/parad/parad_qartod_climatology_test_values.csv:9]: Are these supposed to have a depth parameter/input?

**Review by reedan88 — APPROVED** [review]
Looks good. My only question is about the PARs on the Surface Piercing Profilers - are they supposed to have a depth parameter for the climatology tables? I'm not super familier with the CSPPs.

File: qartod/parad/parad_qartod_climatology_test_values.csv (hunk 1 above)
Comment (cwingard) [reply qartod/parad/parad_qartod_climatology_test_values.csv]: Yes, see newly updated tables.

**Review by wruef — APPROVED** [review]
Looks good, no obvious formatting errors, values look reasonable.

**Pull request merged**